### ISSUE TITLE
Example or Reference for using DigiFont with SSD1306 128x32 OLED ?

### ISSUE THREAD

**Issue by m-r-m-s** [cbm80amiga/DigiFont#3]
Hi @cbm80amiga ,

I saw the [demo](https://www.sciencetronics.com/greenphotons/?p=2211) linked on your YouTube video using the DigiFont library with SSD1306 128x32. I want to implement 7 segment numbers in a similar way using this Library. Do you have an example or reference I could follow to implement DigiFont like in the SSD1306 128x32 demo above? I am unsure what changes would need to be made in the library to make this happen so any help is greatly appreciated. Thank you for all your great work on this library!

**Comment by cbm80amiga** [comment]
I provided many examples for monochrome displays like HX1230, use them as a reference.

**Comment by m-r-m-s** [comment]
> I provided many examples for monochrome displays like HX1230, use them as a reference.

Ok, I will take a look but I am really not advanced in my coding skills yet to know what modifications would need to be made to move from one type of display to the other. I'll give it a try!

I did see there is a link to download a DigiFont SSD1306 128x32 code example here: https://www.sciencetronics.com/greenphotons/?p=2211 I think I missed it the first time I went to the page. Hopefully that will give me some clues.
